WalletWasabi.Fluent/ViewModels/Wallets/Receive/AddressLabelEditViewModel.cs

```diff
@@ -12,8 +13,9 @@ public partial class AddressLabelEditViewModel : DialogViewModelBase<LabelsArray
 {
 	[AutoNotify] private bool _isCurrentTextValid;
 
-	public AddressLabelEditViewModel(IWalletModel wallet, IAddress address)
+	public AddressLabelEditViewModel(UiContext uiContext, IWalletModel wallet, IAddress address)
```

Comment:
I get an error in `ReceiveAddressesViewModel.cs` when I try to use private ctor here. Could you take a look?